media-plugins/mythplugins/mythplugins-31.0.ebuild

```diff
@@ -0,0 +1,165 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 python3_{6,7} )
+PYTHON_REQ_USE="xml"
+
+#BACKPORTS="03f44039848bd09444ff4baa8dc158bd61454079"
+MY_P=${P%_p*}
+
+inherit python-single-r1 readme.gentoo-r1
+
+DESCRIPTION="Official MythTV plugins"
+HOMEPAGE="https://www.mythtv.org"
+# mythtv and mythplugins are separate builds in the same github MythTV/mythtv repository
+SRC_URI="https://github.com/MythTV/mythtv/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+LICENSE="GPL-2+"
+KEYWORDS="~amd64 ~x86"
+SLOT="0/${PV}"
+
+MYTHPLUGINS="mytharchive +mythbrowser mythgame \
+mythmusic mythnetvision +mythnews +mythweather mythzmserver mythzoneminder"
+IUSE="${MYTHPLUGINS} alsa cdda cdr exif fftw +hls ieee1394 libass +opengl raw +theora +vorbis +xml xvid"
+
+# Mythnetvision temporarily disabled by upstream - should be fixed soon.
+REQUIRED_USE="
+	!mythnetvision
+	mytharchive? ( ${PYTHON_REQUIRED_USE} )
+	mythnetvision? ( ${PYTHON_REQUIRED_USE} )
+	mythmusic? ( vorbis )
+	mythnews? ( mythbrowser )
+"
+RDEPEND="
+	dev-libs/glib:2
+	dev-libs/openssl:=
+	dev-qt/qtcore:5
+	dev-qt/qtgui:5
+	dev-qt/qtopengl:5
+	dev-qt/qtsql:5
+	media-libs/freetype:2
+	media-libs/libpng:=
+	sys-apps/util-linux
+	virtual/libudev:=
+	x11-libs/libX11
+	x11-libs/libXext
+	x11-libs/libXinerama
+	x11-libs/libXrandr
+	x11-libs/libXv
+	x11-libs/libXxf86vm
+	alsa? ( media-libs/alsa-lib )
+	hls? (
+		media-libs/faac
+		media-libs/libvpx:=
+		media-libs/x264:=
+		media-sound/lame
+	)
+	ieee1394? (
+		media-libs/libiec61883
+		sys-libs/libavc1394
+		sys-libs/libraw1394
+	)
+	libass? ( media-libs/libass:= )
+	=media-tv/mythtv-${PV}*[alsa?,cdda?,cdr?,exif?,fftw?,ieee1394?,libass?,opengl?,raw?,xml?,xvid]
```

Comment:
I wonder if a ~ would suffice here.

Reply:
Older versions of MythTV often are used for years because each major version generates a new database schema. Both `mythfrontend` and `mythbackend` must use the same database schema to work together. As a result releases are not frequent. 
I may need to make a patch version (i.e. mythtv-31.0_p20nnnnnn.ebuild) from the fixes/XX branch to fix a bug in MythTV.

Now that I have tried to justify my choice I noticed that there is an error in my logic. The database version only changes on major versions. It remains the same for minor versions and the fixes/XX branch. The correct dependency is: `=media-tv/mythtv-${MAJOR_PV}*...` where `MAJOR_PV=31` is near the beginning of the ebuild.

Reply:
Okay, I follow you. But ~ would allow you to make revisions like media-tv/mythtv-SOMEVERSION-r1 and so on, without needing to update all the dependent ebuilds. Maybe the * already has this effect here, though?

Reply:
As I understand it, the `*` allows anything after mythplugins-31 to match as `=`. The documentation specifically notes that `=` is required.